contestant_containers/arch/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM docker.io/library/archlinux
+LABEL maintainer "Trevor Bryant <@apporima>"
+
+# WORKAROUND for glibc 2.33 and old Docker
+# See https://github.com/actions/virtual-environments/issues/2658
+# Thanks to https://github.com/lxqt/lxqt-panel/pull/1562
+RUN patched_glibc=glibc-linux4-2.33-4-x86_64.pkg.tar.zst && \
+    curl -LO "https://repo.archlinuxcn.org/x86_64/$patched_glibc" && \
+    bsdtar -C / -xvf "$patched_glibc"
+
+# Install packages
+RUN pacman -Sy --noconfirm git make gcc pkgconf which python-pip wget hostapd wpa_supplicant bully openssh vim wireshark-cli tmux screen supervisor
```

Comment:
can this more closely match the existing ones?  packages are sometimes differently named across distros but this it not even trying to match :-)

Reply:
Can you elaborate on this? Do you mean _all_ of the other packages that here, but exist in the other contestant containers (hostapd-wpe, etc)?

I used the [Arch Linux - Package Search](https://archlinux.org/packages/) to find the proper package names, as some are different from what we are familiar with. If you know what the other packages are not matching, can you clarify which those are?

Reply:
right now the goal is to install the following packages:
aircrack-ng asleap freeradius-wpe hostapd-wpe iw kismet mdk3 mdk4 pixiewps reaver wifi-honey wifite tshark wireshark termshark vim mlocate man pciutils hashcat wpasupplicant less bash-completion ssh supervisor novnc xvfb x11vnc

plus whatever the distro specific xfce megapackage is, such as "kali-desktop-xfce"

Things like build-essential shouldn't be needed for most players and increases the container size quite a bit.  Assume players who need gcc also know how to install it, etc.

Reply:
Got it. I will find and include all of these excluded packages and push another request.